src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        // createdAt, lastEditedAt, commentGiverName and lastEditorName are required to generate editedCommentDetails
+        data.comment = updatedComment;
+        data.question = logic.getFeedbackQuestion(response.feedbackQuestionId);
+        data.showCommentToString = StringHelper.toString(updatedComment.showCommentTo, ",");
+        data.showGiverNameToString = StringHelper.toString(updatedComment.showGiverNameTo, ",");
+        data.instructorEmailNameTable = instructorEmailNameTable;
```

Comment:
I was doing the same changes, just was a bit reluctant to have a new method with functionality same as existing one.